internal/apis/certmanager/validation/issuer.go

```diff
@@ -389,7 +389,7 @@ func validateVenafiTPPCABundleUnique(tpp *certmanager.VenafiTPP, fldPath *field.
 	}
 
 	if numCAs > 1 {
-		el = append(el, field.Forbidden(fldPath, "may not specify more than one of caBundle/caBundleSecretRef as TPP CA Bundle"))
+		el = append(el, field.Forbidden(fldPath, "may not specify more than one of caBundle/caBundleSecretRef as Certificate Manager, Self-Hosted CA Bundle"))
```

Comment:
Please revert the change. It is preferable to keep `TPP CA Bundle`, as it aligns more closely with the name used in the configuration.